script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -111,17 +129,26 @@ Akiha glares at me.
 [sha:1e6e5834bc92b9e1d7bd9fa48e44a76f51fa7ce8]{
 -- Page 333, Offset 14739.
 -- 翡翠と話しているのがバレるのは当然として、内容まで伝わっているのが怖い。地獄耳かアイツは。
-It's terrifying to me that she understood everything that we said down to its contents as if it was natural to do so. Her ears must be real sharp.
+It's only natural she'd notice me talking to Hisui,
+ but the fact that she could even make out what we were saying is disconcerting.
+ Her hearing must be insanely sharp.
 }
 [sha:ec57cdfcb26a1b085ae902b1847d28c55027b090]{
 -- Page 334, Offset 14740.
 -- 「……もっとも、それも過去の話ですが。
-"...Then again, that is a thing of the past.
+"...Then again, I suppose that too is now a thing of the past.
 }
 [sha:f74e467cf0e1f0f5b191a1cd92965948b33f35af]{
 -- Page 334, Offset 14741.
 -- どうしても話がある、今日一日の出来事を語り合いたい、というのであれば、そのかぎりではありません」
-If you really need to talk, or have something you want to discuss regarding today and such, then there is no problem at all."
+// Not confident on this one.
+// - どうしても話がある
+// - a) "If you must talk about something" (if you cannot bear to be silent) [elected]
```

Comment:
the first one is correct I'm fairly certain. どうしても話がある reads to me more as "if there must be talk" rather than "if there is a topic you must bring up"